### PR TITLE
Add aggregator for LiteHRNet

### PR DESCRIPTION
**Pull request by kprokofi**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Comment by kprokofi** [comment]
This PR removes differences between OTX1.5 and OTX2.0 for LiteHRNet
I checked, that snapshot from OTX1.5 can be loaded now w/o errors

File: src/otx/algo/utils/support_otx_v1.py [hunk 1]
```diff
@@ -97,8 +97,7 @@ def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
         """Load the OTX1.x lite hrnet segmentation checkpoints."""
```
Comment (vinnamkim) [inline src/otx/algo/utils/support_otx_v1.py:97]: @sungmanc, do we still need this function `load_seg_lite_hrnet_ckpt`?

Comment (sungmanc) [reply src/otx/algo/utils/support_otx_v1.py:97]: Now, we could use the `load_common_ckpt` for this function. However, it's still needed to remain due to the different patterns between `SegNext` and `LitHRNet`.

```python3 
@staticmethod
    def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
        """Load the OTX1.x lite hrnet segmentation checkpoints."""
        for key in list(state_dict.keys()):
            val = state_dict.pop(key)
            state_dict[add_prefix + key] = val
        return state_dict
###########################################################
    def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
        """Load the OTX1.x lite hrnet segmentation checkpoints."""
        return OTXv1Helper.load_common_ckpt(state_dict, add_prefix)
```

Comment (vinnamkim) [reply src/otx/algo/utils/support_otx_v1.py:97]: Why not just directly calling `OTXv1Helper.load_common_ckpt(state_dict, add_prefix)` from `LiteHRNet`?

Comment (sungmanc) [reply src/otx/algo/utils/support_otx_v1.py:97]: We also can do what you suggested. I'll patch this part when I check the mobilenetv3 backward compat.

**Review by vinnamkim — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]

**Review by chuneuny-emily — APPROVED** [review]

**Pull request merged**